src/pages/Profile/Profile.test.tsx

```diff
@@ -0,0 +1,39 @@
+import { screen } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import Profile from "./Profile";
+import { randomUserProfiles } from "./mockData";
+import { customRender } from "../../utils/testUtils";
+
+describe("Profile component", () => {
+  it("renders navigation items correctly", () => {
+    customRender(<Profile user={randomUserProfiles[0]} />);
+    expect(screen.getByText(/Name/i)).toBeInTheDocument();
+    expect(screen.getByText(/Bio/i)).toBeInTheDocument();
+    expect(screen.getByText(/Email/i)).toBeInTheDocument();
+    expect(screen.getByText(/Password/i)).toBeInTheDocument();
```

Comment:
Can you separate this block to use variables as an intermediary step, for example:
```
const profileName = screen.getByText(/Name/i);
expect(profileName).toBeInTheDocument();
```
This goes for the other tests in the file as well